blocks/sign-up/sign-up.css

```diff
@@ -0,0 +1,216 @@
+     border-radius: var(--space-24px);
+     background-color: var(--color-white);
+     box-shadow: 0 20px 60px 20px var( --color-darker), 0 2px 4px 0 var(--color-yellowish-white);
+     padding: var(--space-15px) var(--space-35px);
+     width: 26.25rem;
+     position: absolute;
+     top: 50%;
+     right: 0;
+     transform: translateY(-50%);
+ }
+ 
+
+ 
+ .block.sign-up label {
+    display: inline-block;
+    margin-bottom: 0.5rem;
+}
+
+
+ .block.sign-up .signup-container .signup-form-group label {
+     font-size: var(--heading-font-size-ml);
+     color: var(--color-gray-dark);
+     position: relative;
+     padding-bottom: var(--space-15px);
+     margin-bottom: var(--space-10px);
+ }
+ 
+ .block.sign-up .signup-container .signup-form-group label::before {
+     content: "";
+     position: absolute;
+     bottom: 5px;
+     left: 50%;
+     transform: translateX(-50%);
+     width: 100px;
+     height: 3px;
+     background: var(--link-color-orange);
+ }
+ 
+
+
+ .block.sign-up .section.margin {
+     margin: var( --space-45px) 0;
+ }
+ 
+
+
+ .block.sign-up .row {
+     display: flex;
+     flex-wrap: wrap;
+     margin-right: calc(-1 * var(--space-15px));
+     margin-left: calc(-1 * var(--space-15px));
+ }
+
+ .block.sign-up .text-left {
+     text-align: left !important;
+ }
+  
+ .block.sign-up .promotional-text {
+     margin-top: 0.25rem !important;
+     color:var(--color-darker-gray);
+     display: block !important;
+ }
+ 
+ .block.sign-up .signupsections {
+    flex: 0 0 41.6667%;
+    max-width: 41.6667%;
+    position: relative;
+    padding-right:  var(--space-15px);
+    padding-left:  var(--space-15px);
+}
+
+
+
+ .block.sign-up .justify-content-center {
+     justify-content: center !important;
+ }
+ 
+ .block.sign-up .signupbtn {
+    min-height: 3.125rem;
+    font-weight: var(--font-weight-semibold);
+    min-width: 12.25rem;
+    background: var(--link-color-orange);
+    color: var(--color-white);
+    text-transform: uppercase;
+    border: var(--space-1px) solid transparent;
+    border-radius: 1.875rem;
+    padding: var(--space-5px)  var(--space-20px);
+    transition: 0.3s;
+    margin-top:  var(--space-12px) !important;
+     margin-bottom:  var(--space-12px) !important;
+     font-size:1rem;    
+}
+
+
+
+ .block.sign-up input, .block.sign-up .phonenumber-textbox {
+    height: 2.5rem;
+    padding: var(--space-5px) var(--space-15px);
+    color: var(--color-footer-top-note-blue);
+    font-weight: var( --font-weight-small);
+    font-style: italic;
+    border-color: var(--color-lighter-gray-border);
+    font-family: inherit;
+    max-width: 16.25rem;
+    margin: 0 auto var(--space-10px);
+    display: block;
+    width: 100%;   
+    font-size: 1rem;
+    border: 1px solid var( --border-color-lighter-gray);
+    border-radius: 0.25rem;
+}
+
+.block.sign-up input, .block.sign-up .phonenumber-textbox::placeholder {
+    font-size: 1rem;
+}
+
+
+@media (width <= 64rem) {
```

Comment:
We are developing mobile first so the default css should mobile specific and media queries for other screens , so the width should with `>=`

Reply:
fixed